sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Customizations/Models/ConversionErrorCode.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion.Models
+{
+    /// <summary> The ConversionErrorCode. </summary>
+    public readonly partial struct ConversionErrorCode
```

Comment:
Why was this needed as opposed to relying on the code gen?

Reply:
Casing needed to be fixed for a lot of these

Reply:
Fixed through swagger change